UnitTests/CommonTestUtils/SubmoduleTestHelpers.cs

```diff
@@ -33,9 +45,19 @@ public static void UpdateSubmoduleStatusAndWaitForResult(ISubmoduleStatusProvide
             List<DetailedSubmoduleInfo> result = new List<DetailedSubmoduleInfo>();
             provider.StatusUpdated += Provider_StatusUpdated;
 
-            provider.UpdateSubmodulesStatus(
-                workingDirectory: module.WorkingDir,
-                gitStatus: gitStatus);
+            try
+            {
+                ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+                {
+                    await provider.UpdateSubmodulesStatusAsync(
+                        workingDirectory: module.WorkingDir,
+                        gitStatus: gitStatus);
+                });
+            }
+            catch (Exception)
```

Comment:
ditto -- wiith the little difference that `Provider_StatusUpdated` is pointless here.